sass/style.css

```diff
@@ -451,6 +556,32 @@ h3.city {
   .map-search-wrap {
     padding-top: 4px;
   }
+
+  .navbar.ftm {
+    padding: 0;
+  }
+
+  .navbar.ftm .navbar-brand {
+    margin-left: .5rem;
+  }
+
+  .navbar.ftm .navbar-nav .nav-link {
+    text-align: center;
```

Comment:
Centered items look bad in mobile, especially with the dropdown caret being included:

![image](https://user-images.githubusercontent.com/743030/80155883-7ba16a80-8577-11ea-9e18-cecb971e3556.png)

Looks especially bad for the share dropdown. The icons should be aligned:

![image](https://user-images.githubusercontent.com/743030/80155926-8f4cd100-8577-11ea-870b-91722324b441.png)

Extra ick:

![image](https://user-images.githubusercontent.com/743030/80155942-9c69c000-8577-11ea-90b9-6e378441fc03.png)

Reply:
Hm... we thought to center these links because accessibility would be better if a person doesn't have to reach all the way across to the left of the screen with their thumb, if they are right handed and have small and/or arthritic hands.

Reply:
That shouldn't be a problem. The full width is clickable.

Reply:
I know, but our users don't know :) People naturally tap/click on what they can see, not what's invisible (ref: years of user testing folks from all walks of life)

Reply:
Hmm. maybe there's some affordance we can use to indicate the whole row is clickable, instead of introducing the weirdness of misaligned icons?

Reply:
I hear you and we'll keep iterating on the design, but for now I'm going to go ahead and merge! Don't let perfect be the enemy of the good :)